### ISSUE TITLE
Risky Repository Name

### ISSUE THREAD

**Issue by ian13456** [michaljaz/webmc#95]
Hi, great work you got here! 

Just a quick reminder to maybe change the name of the repository to something unrelated to Minecraft unless you've made sure Microsoft is fine with it. I had a project of a web-based Minecraft two years ago too, but when it got too much attention, Microsoft struck it down with a DMCA copyright strike... I spent six months on [this project](https://github.com/ian13456/mc.js) only to see it taken down simply because Microsoft wasn't happy with the name "mc", and said that I was using texture packs related to Minecraft, I was devastated. Also, Microsoft also claimed that I was "fishing for real Minecraft accounts", even though my project was nowhere deployed online. So you might want to watch out for that aspect too.

In conclusion, I'm here to congratulate you on the good work you've put into this, and also to remind you that maybe it's not a good idea to keep the name "Minecraft" as the project name. If you don't agree with me, feel free to close this issue! :raised_hands:

**Comment by michaljaz** [comment]
You' re right, but hmm... what name

**Comment by michaljaz** [comment]
Maybe webmc

**Comment by michaljaz** [comment]
Ok changed :)

**Comment by michaljaz** [comment]
changed :)

**Comment by ian13456** [comment]
BTW, maybe also swap out the textures

> On top of that, the download is being advertised with copyrighted Minecraft imagery and textures, acting as the backdrop to the site, without Microsoft’s consent or authorization. This is not an action protected by any fair use doctrine.

That was from my DMCA takedown notice.
